lib/src/gotrue_api.dart

```diff
@@ -214,8 +214,16 @@ class GoTrueApi {
       if (response.error != null) {
         return GotrueSessionResponse(error: response.error);
       } else {
-        final session =
+        Session session =
             Session.fromJson(response.rawData as Map<String, dynamic>);
+        if (session.user == null) {
+          final userResponse = await getUser(session.accessToken);
+          if (userResponse.user != null) {
+            session = session.copyWith(user: userResponse.user);
+          } else {
+            // print(userResponse.error);
+          }
```

Comment:
Do we need this else statement here?

Reply:
No

Reply:
It might not be a bad idea to print `userResponse.error` here though as you have it in the comments. I like that idea, but if you have a better idea of how to handle the situation, I would love to hear it as well!